Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.0
+import PackageDescription
+
+let package = Package(
+    name: "KWStepper",
+    platforms: [
+        .iOS(.v10)
```

Comment:
`KWStepper` works with iOS 8 deployment targets, which are still supported in Xcode 11.

```suggestion
        .iOS(.v8)
```